### PR TITLE
add cron file for volunteer list

### PR DESCRIPTION
**Pull request by Imran-yousuf**
**PLEASE NOTE:**

This is only a issue tracker for issues related to the CakePHP Application Skeleton.
For CakePHP Framework issues please use this [issue tracker](https://github.com/cakephp/cakephp/issues).

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

The best way to propose a feature is to open an issue first and discuss your ideas there before implementing them.

Always follow the [contribution guidelines](https://github.com/cakephp/cakephp/blob/master/.github/CONTRIBUTING.md) guidelines when submitting a pull request. In particular, make sure existing tests still pass, and add tests for all new behavior. When fixing a bug, you may want to add a test to verify the fix.

### CODE REVIEW

File: src/Command/BhaktasCommand.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        parent::initialize();
+        $this->loadModel('Bhaktas');
+    }
+    public function execute(Arguments $args, ConsoleIo $io)
+    {
+		$bhaktas = $this->Bhaktas->find()
+            ->select(['Bhaktas.id', 'Bhaktas.nev_szuletesi', 'services.szolgalat_kezdete'])
+            ->where([
+                'Bhaktas.communityrole_id' => BhaktasCommand::ONKENTES_TAMOGATO,
+                'services.szolgalat_kezdete <' => new Time('5 years ago')
```
[the remark this answers is not in the document]
Comment (Imran-yousuf) [reply src/Command/BhaktasCommand.php]: should I create a const or something like that?

**Comment by Imran-yousuf** [comment]
just convert it in Const

File: src/Command/BhaktasCommand.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (rrd108) [reply src/Command/BhaktasCommand.php]: No, shell scripts can take console arguments
https://book.cakephp.org/3/en/console-and-shells/commands.html#defining-arguments-and-options

[the remark this answers is not in the document]
Comment (Imran-yousuf) [reply src/Command/BhaktasCommand.php]: Got it, please have a look.

**Pull request closed without merging**